src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -193,7 +210,10 @@ class PillSelectorDropdown extends React.Component<Props, State> {
                     <Label text={label}>
                         <PillSelector
                             onChange={noop} // fix console error
+                            onCompositionEnd={this.onCompositionEnd}
+                            onCompositionStart={this.onCompositionStart}
```

Comment:
Do these need to be up here or can they live below `{...inputProps}` with the other event handler props?

Reply:
Put here so that if someone is overriding them via inputProps that continues as today